_posts/2025-04-25.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Upcoming changes with v20.14.0
+severity: 1
```

Comment:
Severity should be lowered, probably 4 is more closer to what it should be (compared to other posts)

Reply:
Its closed.